etc/profile-a-l/loupe.profile

```diff
@@ -0,0 +1,50 @@
+# Firejail profile for loupe
+# Description: GNOME's modern Image Viewer program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include loupe.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.local/share/Trash
+noblacklist ${HOME}/.Steam
+noblacklist ${HOME}/.steam
+
+#include disable-common.inc
```

Comment:
@glitsj16 @SkewedZeppelin

This profile has no disable-common and no `whitelist` or `read-only` in
`${HOME}`.

I can modify ~/.bashrc with it.

So what is the point?

If editing images is not a common use case, I'd add `read-only ${HOME}`.

If not, I'd uncomment disable-common.

Reply:
I couldn't get disable-common to work

It is still on my TODO list

Reply:
> I couldn't get disable-common to work nothing stuck out when I strace'd it I
> think it is a conflict with how it invokes bwrap
> 
> also simple edits like rotation is common task to not want ro
> 
> It is still on my TODO list

I see.

Then I'd at least add a warning to the profile:

```diff
 # Persistent global definitions
 include globals.local
+
+# Warning: This profile is currently WIP and is not very secure.
```

And maybe remove it from firecfg.config since it's still WIP, to avoid passing
a false sense of security to users.

Note that you can add programs to firecfg in /etc/firejail/firecfg.d/ now (see
#5876).

> edit: also the primary reason I made a profile for it is because when
> hardened_malloc is used images never load due to the rlimit set on its
> sandbox which isn't enough virtual memory for the hmalloc to function

> my typical
> [workaround](https://github.com/divestedcg/rpm-hardened_malloc?tab=readme-ov-file#known-issues)
> is blacklisting /etc/ld.so.preload in my builds of firejail in my rpm repo
> and I saw loupe didn't have a profile yet

That clarifies things a bit; thanks for the details.